node_modules/gh-orgunit/tests/test-orgunit.js

```diff
@@ -616,6 +628,89 @@ describe('Organisational units', function() {
+                assert.strictEqual(serieBResult.canManage, canManageSerieB);
+                return callback();
+            });
+        };
+
+        /**
+         * Test that verifies the canManage property on series
+         */
+        it('verify the canManage property on series', function(callback) {
+            // Create some test data
+            OrgUnitTestUtil.assertCreateOrgUnit(global.tests.admins.cam2014.client, 'Parent', 'part', null, function(parent) {
+                OrgUnitTestUtil.assertCreateOrgUnit(global.tests.admins.cam2014.client, 'Module', 'module', {'parent': parent.id}, function(module) {
+
+                    // Generate 3 users
+                    TestsUtil.generateTestUsers(global.tests.apps.cam2014, 3, false, function(simon, nico, sam) {
+
+                        // Simon and Nico create a serie
+                        var start = moment().subtract(30, 'days').format();
+                        var end = moment().add(30, 'days').format();
+                        SeriesTestUtil.generateSerieWithEvents(simon.client, 1, 5, start, end, function(simonSeries) {
+                            SeriesTestUtil.generateSerieWithEvents(nico.client, 1, 5, start, end, function(nicoSeries) {
+
+                                // Add each serie to the module
+                                var serieIds = [ simonSeries[0].id, nicoSeries[0].id ];
+                                OrgUnitTestUtil.assertAddOrgUnitSeries(global.tests.admins.cam2014.client, module.id, serieIds, function() {
+
+                                    // Anonymous users can't manage any series
+                                    TestsUtil.getAnonymousAppUserClient(global.tests.apps.cam2014, function(anonymousAppClient) {
+                                        expectSeries(anonymousAppClient, parent, module, simonSeries[0], nicoSeries[0], false, false, function() {
+
+                                            // Sam can't manage any series
+                                            expectSeries(sam.client, parent, module, simonSeries[0], nicoSeries[0], false, false, function() {
+
+                                                // Simon can only manage his serie
+                                                expectSeries(simon.client, parent, module, simonSeries[0], nicoSeries[0], true, false, function() {
+
+                                                    // Nico can only manage his serie
+                                                    expectSeries(nico.client, parent, module, simonSeries[0], nicoSeries[0], false, true, function() {
+
+                                                        // An app admin can manage both
+                                                        expectSeries(global.tests.admins.cam2014.client, parent, module, simonSeries[0], nicoSeries[0], true, true, function() {
+
+                                                            // A global admin can manage any series
```

Comment:
In my defence, I wouldn't have made the comment if there weren't any other comments :p